### PR TITLE
Add support for Broadband Network Platform Software from GCOM Technologies Co.,Ltd.

### PR DESCRIPTION
**Pull request by freddy36**

### CODE REVIEW

File: docs/Supported-OS-Types.md [hunk 1]
```diff
@@ -76,13 +76,17 @@
    * [XOS](/lib/oxidized/model/xos.rb)
  * F5
    * [TMOS](/lib/oxidized/model/tmos.rb)
+ * Fiberstore
```
Comment (laf) [inline docs/Supported-OS-Types.md]: How come this is in here?

File: lib/oxidized/model/gcombnps.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    end
+  end
+
+  cfg :telnet, :ssh do
+    pre_logout 'exit'
+  end
+
+end
+
+# vim: set noai ts=2 sw=2:
```
Comment (laf) [inline lib/oxidized/model/gcombnps.rb]: Can you remove these two lines please.

File: lib/oxidized/model/gcombnps.rb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  cmd 'show running-config' do |cfg|
+    cfg
+  end
+
+  cmd 'show interface sfp' do |cfg|
+    cfg = cfg.each_line.to_a
+
+    cfg = cfg
+      .reject { |line| line.match /^  Temperature/ }
```
Comment (laf) [inline lib/oxidized/model/gcombnps.rb]: A lot of the other models do it this way:

https://github.com/ytti/oxidized/blob/master/lib/oxidized/model/aireos.rb#L45

Can you switch to that or was it done like this for a reason?

File: docs/Supported-OS-Types.md (hunk 1 above)
Comment (freddy36) [reply docs/Supported-OS-Types.md]: GCOM also produces OEM hardware for other companies
The Fiberstore S3800 is one switch I'm aware of.
I added it so other users can easily find the correct module.

Comment (laf) [reply docs/Supported-OS-Types.md]: Ok, makes sense.

**Comment by freddy36** [comment]
requested changes have been made

**Pull request merged**